vm/cvm_tracing/src/lib.rs

```diff
@@ -25,9 +26,9 @@ pub const CVM_ALLOWED: Empty = Empty;
 /// not be logged out of a confidential environment.
 pub const CVM_CONFIDENTIAL: Empty = Empty;
 
-/// A tracing filter that will only allow events that are marked as [`CVM_ALLOWED`].
+/// A tracing filter that will not allow events that are marked as [`CVM_CONFIDENTIAL`].
 pub fn confidential_event_filter<S: Subscriber>() -> impl Filter<S> {
-    FilterFn::new(move |m| m.fields().field("CVM_ALLOWED").is_some())
+    FilterFn::new(move |m| m.fields().field("CVM_CONFIDENTIAL").is_some().not())
```

Comment:
Definitely not. The design here was intentional. We can not allow un-audited tracing statements to be logged.

Reply:
The problem is all the tracing statements that already exist that may log sensitive or confidential information. The auditing needs to be opt-in on a per-statement level. I agree that this would be easier, but it is also an unacceptable level of risk to meet our CVM confidentiality guarantees.